src/data/manifest.rs

```diff
@@ -81,12 +80,9 @@ impl Manifest {
     }
 
     /// add a dependency
-    pub fn add_dependency(&mut self, dependency_name: &str, dependency_version: &str) {
-        let dependencies = self.dependencies.get_or_insert(BTreeMap::new());
-        dependencies.insert(
-            dependency_name.to_string(),
-            Value::String(dependency_version.to_string()),
-        );
+    pub fn add_dependency(&mut self, dependency_name: String, dependency_version: String) {
+        let dependencies = self.dependencies.get_or_insert(HashMap::new());
```

Comment:
⛳️ 
`         let dependencies = self.dependencies.get_or_insert(Default::default()); `

avoids needing to change the line in cases like this